inject.js

```diff
@@ -152,7 +154,26 @@
         }
       }.bind(this));
 
-      target.playbackRate = tc.settings.speeds[target.src];
+      var observer=new MutationObserver((mutations)=> {
+        mutations.forEach((mutation) => {
+          if (mutation.type === 'attributes' && mutation.attributeName === 'src'){
+            var controller = document.querySelector(`div[data-vscid="${this.id}"]`);
+            if(!controller){
+              return;
+            }
+            if (!mutation.target.src) {
+              controller.classList.add('vsc-nosource');
+              console.log('Hiding: no source');
```

Comment:
Are the console.log's necessary? 

Sidenote: we should probably keep them around, as they can be helpful in debugging, but we should probably make a helper function (separate PR) that checks for a debug flag or some such, that can be set in development but does not otherwise need to fire for all users.

Reply:
Logs clearly not necessary. Forgot to take them out from  debugging and was gonna commit again to remove them but thought maybe they were worth keeping as they shouldn't fire too often. Probably not worth keeping without  a debug logger function.